tests/SprykerTest/Zed/EventBehavior/_support/Helper/EventBehaviorHelperTrait.php

```diff
@@ -7,8 +7,13 @@
 
 namespace SprykerTest\Zed\EventBehavior\Helper;
 
+use Codeception\Module;
```

Comment:
same question as in `spryker/spryker`: the issue repeats in each file using  `ModuleHelperTrait`, if it's used can it be moved directly to the trait then?

Reply:
Yes, You are right!